package.json

```diff
@@ -67,6 +71,7 @@
     "@types/js-yaml": "^4.0.4",
     "@types/micromatch": "^4.0.2",
     "jest": "^27.0.6",
+    "npm-run-all": "^4.1.5",
```

Comment:
It should already been installed by my `@1stg/common-config` like `eslint`, are they unavailable?

Reply:
As you are directly using the package binary you should install it. You cannot rely on indirect dependencies especially in yarn v2 and above (it throws errors).

Reply:
Then `eslint` will error too?

Reply:
no problem I will remove this dependency.